<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMainWindow</class>
 <widget class="QMainWindow" name="LoginMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Titleframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>662</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 242, 246);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/photo01/image/LoginAndRegister/toast_close_black.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MinWindowButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="MaxWindowButton">
     <property name="geometry">
      <rect>
       <x>605</x>
       <y>9</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <property name="text">
      <string>口</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="titleLable">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;跃渊系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="mayiLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/LoginAndRegister/蚂蚁.png);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Themewidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>76</y>
      <width>662</width>
      <height>366</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>33</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;欢迎登陆跃渊系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="usernametextEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>86</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="passwordtextEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="remberpasswordcheckBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="autologincheckBox">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="frgetpasswordlabel">
     <property name="geometry">
      <rect>
       <x>563</x>
       <y>210</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>忘记密码？</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>290</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 168, 255);
color: rgb(255, 255, 255);
background-color: rgb(6, 168, 255);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QLabel" name="scanlabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>扫一扫</string>
     </property>
    </widget>
    <widget class="QLabel" name="QRcodelabel">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>100</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/LoginAndRegister/二维码.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="usernameframe">
     <property name="geometry">
      <rect>
       <x>341</x>
       <y>90</y>
       <width>38</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/LoginAndRegister/username.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="passwordframe">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>154</y>
       <width>36</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/LoginAndRegister/Lock.jpg);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordlineEdit2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>151</y>
       <width>249</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernamelineEdit2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>87</y>
       <width>249</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="registerNewUserPushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>325</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注册新账号</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="UsernameAndPasswordTipLable">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>290</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
